### ISSUE TITLE
Outline of milestone 1 scripts

### ISSUE THREAD

**Issue by talhaadnan100** [UBC-MDS/Breast-Cancer-Prediction#8]
Putting down my initial thoughts for the milestone 1 scripts in the context of our project. We can refine these when we meet in person, and strategize how we tackle them.

1. First script that reads in some data and saves a clean/tidy version: 
  * assert statements that verify all variables are complete (i.e. don't have NAs or missing values).
  * assert statement that verifies variable type to be numeric and categorical classification to be integer.
2. Second script that reads the data from the first script and creates an exploratory visualization: 
  * Visualize distributions of training data features using histograms to identify better predictors from our data set
3. Third script that performs ML supervised learning classification:
  * split data set into 80% for building the model and 20% for final predictions
  * use Cross Validation to pick best hyperparameter (`max_depth` and `min_samples_split`)
  * run model on the best selected hyperparameters (`max_depth` and `min_samples_split`)
 and perform predictions on the 20% data set
4. Fourth script presents the results in .ipynb:
  * report accuracy of predictions
  * visualize the data on which inaccurate predictions were made

**Comment by talhaadnan100** [comment]
@akanshaVashisth, please add another check in script one that ensures data is ordered in the manner that features are to the left and `Classification` is to the right. This can create X and y in script #3 without hardcoding feature column names. Thanks!

**Comment by akanshaVashisth** [comment]
Achieved the goal of this milestone (b0498304a4cc72236588013e1388ced21cbc31aa)
